src/features/auth/AuthContext.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext, useState } from 'react'
+
+import { clearRefreshToken, saveRefreshToken } from 'libs/keychain'
+import { saveAccessToken, clearAccessToken } from 'libs/storage'
+
+import { signin } from './api'
+
+interface SignInBody {
+  email: string
+  password: string
+}
+
+export interface IAuthContext {
+  loggedIn: boolean
```

Comment:
J'ai une préférence pour les variables booléennes qui sont écris en formule interrogative 
```suggestion
  isLoggedIn: boolean
```